extensions/cobalt/README.md

```diff
@@ -0,0 +1,53 @@
+<div align="center">
+    <img width="100" src="./media/icon.png">
```

Comment:
You can use images from metadata here, no need to have the image in media if it's the same as a metadata